aws/components/externalnetwork/setup.ftl

```diff
@@ -24,14 +24,45 @@
             [#continue]
         [/#if]
 
+        [#list solution.Alerts?values as alert ]
+
+            [#local monitoredResources = getMonitoredResources(core.Id, resources, alert.Resource)]
+            [#list monitoredResources as name,monitoredResource ]
+
+                [@debug message="Monitored resource" context=monitoredResource enabled=false /]
+
+                [#switch alert.Comparison ]
+                    [#case "Threshold" ]
+                        [@createAlarm
+                            id=formatDependentAlarmId(monitoredResource.Id, alert.Id )
+                            severity=alert.Severity
+                            resourceName=core.FullName
+                            alertName=alert.Name
+                            actions=getCWAlertActions(occurrence, solution.Profiles.Alert, alert.Severity )
+                            metric=getMetricName(alert.Metric, monitoredResource.Type, core.ShortFullName)
+                            namespace=getResourceMetricNamespace(monitoredResource.Type, alert.Namespace)
+                            description=alert.Description!alert.Name
```

Comment:
would `name` be a useful alternative here, as it's more meaningful to the user having come from their Solution ?
```suggestion
                            description=alert.Description!name
```